### PR TITLE
Fix zero seconds handling

### PR DESCRIPTION
**Pull request by CryptoAssassin1**
## Summary
- handle `0` seconds in `convertSecondsToMinutesOrHours`
- add test covering zero seconds case
- add changeset entry

## Testing
- `npm test` *(fails: EHOSTUNREACH)*

### CODE REVIEW

File: packages/widget/src/utils/number.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export function calculatePercentageChange(
 }
 
 export const convertSecondsToMinutesOrHours = (seconds?: number) => {
-  if (!seconds) {
+  if (seconds === undefined) {
```
Comment (codingki) [inline packages/widget/src/utils/number.ts:33]: it making it only return when the value is undefined
